ui/ozone/platform/wayland/wayland_window.h

```diff
@@ -112,6 +117,10 @@ class WaylandWindow : public PlatformWindow, public PlatformEventDispatcher {
   bool has_pointer_focus_ = false;
   bool has_keyboard_focus_ = false;
 
+  bool is_minimized_ = false;
```

Comment:
Not really, as long as when one restores a window after the minimize state, we check if there has been the maximize state and restore it back.